zkdb/src/storage-engine/ipfs.ts

```diff
@@ -101,10 +102,11 @@ export const newLibP2p = async (
   const config: Libp2pOptions = {
     services: {
       dht: kadDHT({ allowQueryWithZeroPeers: true }),
+      identify: identifyService(),
```

Comment:
Wow, will this do the trick?

Reply:
Yes, it looks like a peer was not share the information about themselves with the network. identifyService helps establish connections between peers and enables peers in the network to share and update information about themselves

Reply:
I do the test and let you know, awesome work mate.